src/app/doctors/[id]/page.module.css

```
@@ -0,0 +1,438 @@
+              .ratingContent {
+                display: flex;
+                align-items: center;
+                gap: 0.5rem;
+                border: 1px solid var(--color-gray-70);
+                border-radius: var(--border-radius);
+                padding: 0.1rem 0.3rem;
+                background-color: var(--color-gray-70);
+                margin: 0 0 0 0.5rem;
+              }
+
+              .average {
+                margin: 0 0.2rem 0 0.5rem;
+                color: var(--color-primary);
+              }
+
+              .totalVotes {
+                color: var(--color-gray-10);
+              }
+            }
+          }
+        }
+
+        .description {
+          padding: 1rem 0;
+          margin-top: 1rem;
+
+          & h3 {
+            font-weight: 600;
+          }
+
+          .descriptionContent {
+            border-radius: var(--border-radius);
+            border: 1px solid var(--color-gray-80);
+            background-color: light-dark(
+              var(--color-gray-98),
+              var(--color-gray-40)
+            );
+            margin-bottom: 1rem;
+            box-shadow: 0 0 10px var(--color-gray-10);
+            display: flex;
+            flex-direction: column;
+            gap: 1rem;
+
+            padding: 1rem 1rem 1rem 0;
+            margin-top: 0.5rem;
+          }
+        }
+
+        .activity {
+          padding: 1rem 0;
+          margin-top: 1rem;
+
+          & h3 {
+            font-weight: 600;
+          }
+          .activityContent {
+            border-radius: var(--border-radius);
+            border: 1px solid var(--color-gray-80);
+            background-color: light-dark(
+              var(--color-gray-98),
+              var(--color-gray-40)
+            );
+
+            margin-bottom: 1rem;
+            box-shadow: 0 0 10px var(--color-gray-10);
+            display: flex;
+            flex-direction: column;
+            gap: 1rem;
+
+            padding: 1rem;
+            margin-top: 0.5rem;
+
+            .activity_card {
+              margin-top: 0.5rem;
+              display: flex;
+              position: relative;
+              flex-direction: row;
+              width: 100%;
+              height: auto;
+              max-width: 100%;
+              border: 1px dashed
+                light-dark(var(--color-gray-10), var(--color-gray-90));
+              min-width: 0;
+              border-radius: 0.5rem;
+              padding: 1rem;
+
+              .activity_info {
+                display: flex;
+                flex: 1;
+                align-self: stretch;
+                pointer-events: none;
+
+                & svg {
+                  margin-left: 0.5rem;
+                  position: relative;
+                  -o-object-fit: cover;
+                  object-fit: cover;
+                  max-width: 100%;
+                  width: 24px;
+                  height: 24px;
+                  min-width: 24px;
+                  flex-shrink: 0;
```

اکثر اینا الکی استفاده شدن

```suggestion
                  margin-left: 0.5rem;
                  font-size: 1.5rem;
```